satpy/readers/hrit_jma.py

```diff
@@ -265,6 +275,10 @@ def get_dataset(self, key, info):
         # Calibrate and mask space pixels
         res = self._mask_space(self.calibrate(res, key.calibration))
 
+        # Add scanline acquisition time
+        res['acq_time'] = ('y', self.acq_time)
```

Comment:
Is this adding the acquisition time as a coordinate ? (which is good)

Reply:
Will change it to `res.coords['acq_time'] = ...` to be more explicit